packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -1473,12 +1473,19 @@ function pushTitleImpl(
   }
   target.push(endOfStartTag);
 
-  const child =
-    Array.isArray(children) && children.length < 2
-      ? children[0] || null
-      : children;
-  if (typeof child === 'string' || typeof child === 'number') {
-    target.push(stringToChunk(escapeTextForBrowser(child)));
+  const child = Array.isArray(children)
+    ? children.length < 2
+      ? children[0]
+      : null
```

Comment:
this would cause `<svg><title>hello {"world"}</title></svg>` to render `null` if enableFloat is true and `hello <!-- -->world` if enableFloat is false. Is it ok to put these same restrictions on svg titles?

Reply:
that doesn't change with this PR. that was already happening because it was setting child to be an array which is an object and so is ignored.

The inconsistencies already exist.